developer/src/common/web/utils/src/types/ldml-keyboard/ldml-keyboard-xml.ts

```diff
@@ -38,7 +38,7 @@ export interface LKImport {
   /**
    * import base, currently `cldr` is supported
    */
-  base: string;
+  base?: string;
```

Comment:
Should we make this:

```suggestion
  base?: 'cldr';
```

Reply:
May need `base?: 'cldr' || '';`?

Reply:
| but yeah